assignments/objects.js

```diff
@@ -19,46 +20,125 @@ const example = {
 
 // Write your intern objects here:
 
+const michael = {
+    "id": 0,
+    "name": "Michael",
+    "email": "email@email.com",
+    "gender": "M"
+  }
+  const Mitzi = {
+    "id": 1,
+    "name": "Mitzi",
+    "email": "mmelloy0@psu.edu",
+    "gender": "F"
+  }
+  const Kennan = {
+    "id": 2,
+    "name": "Kennan",
+    "email": "kdiben1@tinypic.com",
+    "gender": "M",
+    "speak": function(){
+      console.log("Hello, my name is " + this.name)
+    }
```

Comment:
Your speak method works fine being defined here, but don't forget that you can also define a method outside of the object, like so...
`kennan.speak = function() {
  return "Hello, my name is Kennan!";
}`